deployment/hasura/metadata/databases/tables/actions/action_definition.yaml

```
@@ -0,0 +1,47 @@
+table:
+  name: action_definition
+  schema: actions
+configuration:
+  custom_name: "action_definition"
+object_relationships:
+  - name: workspace
+    using:
+      foreign_key_constraint_on: workspace_id
+  - name: uploaded_file
+    using:
+      foreign_key_constraint_on: action_file_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
```

Major: Column list should not be `"*"`, as we do not, for example, want the user to specify `id`. I'd recommend the following column list:

```suggestion
      columns: [name, description, action_file_id, workspace_id]
```

`parameter_schema` and `settings_schema` will be set by the server after insertion. `settings` requires `settings_schema` to be set before it can have a value. `owner` and `updated_by` are set by the `set` block on L34, and `created_at` and `updated_at` will be prefilled with the current time.